unlock-app/src/__tests__/services/wedlockService.test.js

```diff
@@ -3,7 +3,7 @@ import WedlockService, { emailTemplate } from '../../services/wedlockService'
 
 jest.mock('axios')
 
-let w
+let w = new WedlockService('http://notareal.host')
```

Comment:
Instantiating this variable to begin with allows most editors to infer the type and do completion on the object.